docs/templates/privacy/1.0.gohtml

```diff
@@ -0,0 +1,26 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <title>Matrix.org Privacy policy</title>
```

Comment:
We may want to populate this file with more neutral server names / wording, i.e. not `matrix.org`.